trino-connector/src/main/java/org/apache/gravitino/trino/connector/GravitinoConfig.java

```diff
@@ -69,6 +69,13 @@ public class GravitinoConfig {
           "true",
           false);
 
+  private static final ConfigEntry GRAVITINO_CLUSTER =
+      new ConfigEntry(
+          "gravitino.cluster",
+          "The cluster name of the trino, if it's empty, all the catalogs handled by locally",
```

Comment:
`all the catalogs handled by locally` ->  `all catalogs will be handled locally`